cache/edge_lambdas/src/redirects.ts

```diff
@@ -15,12 +15,9 @@
  *    - User needs to remember to write it down
  *    - Print to digital transmission
  *
- * See https://app.gitbook.com/o/-LumfFcEMKx4gYXKAZTQ/s/DPDDj27NI2F2kPukWrC1/developer-handbook/urls-on-wellcomecollection.org
```

Comment:
@LaurenFBaily this page was 404ing, I don't know if you'd like to review this whole comment and add a different reference link?